### PR TITLE
Update doc version after 140 release

### PR DESCRIPTION
**Pull request by greg-fer**
**To be merged as the first PR after release**

### CODE REVIEW

File: doc/versions.json [hunk 1]
```diff
@@ -13,9 +14,15 @@
   ],
   "COMPONENTS_BY_VERSION": {
     "latest": {
-      "ncs": "1.3.99",
-      "nrfxlib": "1.3.99",
-      "zephyr": "2.3.99",
+      "ncs": "1.4.99",
+      "nrfxlib": "1.4.99",
+      "zephyr": "2.4.99",
```
Comment (greg-fer) [inline doc/versions.json:19]: @ru-fu , should we use 2.4.99 here? The upmerge is planned after 1.4.0 release...

File: doc/nrf/release_notes.rst [hunk 2]
```diff
@@ -5,19 +5,20 @@ Release Notes
 
 See the release notes for information about specific |NCS| releases.
 
-Important issues that are discovered after a release is tagged are listed in the `Known issues`_ wiki page.
+Important issues that are discovered after a release is tagged are listed on the :ref:`known_issues` page.
 
 .. important::
    A "99" at the end of the version number of this documentation indicates continuous updates on the master branch since the previous major.minor release.
-   Changes between releases that are merged into the master branch are tracked in the `nRF Connect SDK changelog`_ in the sdk-nrf repository wiki.
+   Changes between releases that are merged into the master branch are tracked on the :ref:`ncs_release_notes_latest` page.
```
Comment (ru-fu) [inline doc/nrf/release_notes.rst:12]: This depends on the decision where we want to keep the changes, same as the changes to the release-notes-latest file.

File: doc/versions.json (hunk 1 above)
Comment (ru-fu) [reply doc/versions.json]: No, that should be updated with the upmerge. 
@carlescufi Is there some checklist to make sure to update this file as well?

Comment (carlescufi) [reply doc/versions.json]: Correct, this should be left at 2.4.0, which is the current zephyr version. On the next upmerge this needs to be updated. I will add a reminder for this in the upmerge page.

File: doc/nrf/release_notes.rst (hunk 2 above)
Comment (greg-fer) [reply doc/nrf/release_notes.rst]: Decision will be taken after release.

**Review by pdunaj — APPROVED** [review]

**Review by ru-fu — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**